CMakeLists.txt

```diff
@@ -129,8 +133,10 @@ set(DYNAMIC_LIB
     dl
     z
     )
+set(BRPC_PRIVATE_LIBS "-lgflags -lprotobuf -lleveldb -lprotoc -lrt -lssl -lcrypto -ldl -lz")
```

Comment:
missing pthread?

Reply:
I intensionlly left pthread out because pthread flags can vary by platforms and I think it would be better to let users find thread library using their own build system, like find_package(Threads). Anyway by linking against brpc developers should expect their program to be multi-threaded, shouldn't they?